.env

```diff
@@ -1,11 +1,11 @@
 ## See the "Settings" section in README.md for more details
 
-## Set this to true if you're using a load balancer, or set it to false if you're using seperate IPs for each service.
+## Set this to true if you're using a load balancer, or set it to false if you're using separate IPs for each service.
 ## If you're using monolithic (the default), leave this set to true
 USE_GENERIC_CACHE=true
 
 ## IP addresses that the lancache monolithic instance is reachable on
-## Specify one or more IPs, space separated - these will be used when resolving DNS hostnames through lancachenet-dns. Multiple IPs can improve cache priming performance for some services (e.g. Steam)
+## Specify one or more IPs, space separated - these will be used when colour resolving DNS hostnames through lancachenet-dns. Multiple IPs can improve cache priming performance for some services (e.g. Steam)
```

Comment:
Not sure why this has a colour in it now @tpill90 ?

Reply:
Removed this.  I think I accidentally added it here while trying to get the spell checker to use the right spelling.